storage/flux/reader.go

```diff
@@ -819,6 +601,234 @@ func groupKeyForGroup(kv [][]byte, spec *query.ReadGroupSpec, bnds execute.Bound
 	return execute.NewGroupKey(cols, vs)
 }
 
+type windowAggregateIterator struct {
```

Comment:
Okay, so you moved all of this code? That makes it difficult to see what changed. Did you have to change a lot?

Reply:
My bad, I did not realize I changed the structure of this file. This file is a complete copy and paste. I did not make any manual edits.